### PR TITLE
Added storage to save latest type

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR https://github.com/sulu/sulu/pull/3107

This PR breaks the compatability to sulu 1.4 how should we handle that? @danrot @alexander-schranz @chirimoya

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@wachterjohannes create a polyfill for this case (dont know how much work) or remove `~1.4` from the composer.json.

**Comment by wachterjohannes** [comment]
i have added a small polyfill which will be registered if the missing module is not loaded.

File: Resources/public/js/main.js [hunk 1]
```diff
@@ -18,6 +18,15 @@ require.config({
     }
 });
 
+if (!require.defined('services/husky/storage')) {
+
```
Comment (alexander-schranz) [inline Resources/public/js/main.js]: @wachterjohannes we should add a comment here so we now when sulu article bundle requires 1.5 we can remove this polyfill.

**Comment by wachterjohannes** [comment]
@alexander-schranz fixed

**Comment by danrot** [comment]
Maybe polyfill is the wrong word, because it doesn't implement the functionality at all. But that is nitpicking 😃

**Pull request merged**